src/components/editions/headerMedia/index.tsx

```diff
@@ -115,14 +80,13 @@ const getImageStyle = (
 	return css`
 		display: block;
 		width: 100%;
-		height: calc(100vw * ${height / width});
 
 		${from.tablet} {
-			width: ${tabletImageWidth}px;
-			height: ${(tabletImageWidth * height) / width}px;
+			width: calc(100vw - 3.75rem);
```

Comment:
Nice one! 👍
Don't know how that happened again since we fixed it once already [here](https://github.com/guardian/apps-rendering/pull/1133/files#r567728860) .. we didn't have storyshots back then so hopefully next time we'll catch it in time 🤞